### PR TITLE
💄Update success UX

### LINKED ISSUE #733: 💄 Enquiry form - Success UX is weird
https://www.loom.com/share/57484f6a13474b7abeac7597260ca177

### Current flow
1. Click submit 
2. See green success message

Image

3. Get redirected to Thank You page

### Recommended solution
1. Click submit
2. Popup closes
3. Success toast

### PR DESCRIPTION
**Pull request by KristenHu**
closed #733: Update success UX to submitted modal 

Affected routes:

- `/consulting/remote-work` etc.

image

### CODE REVIEW

**Comment by jaydenalchin** [comment]
Hi @KristenHu thanks for getting onto this so quickly! Here's my feedback: 

- [ ] Modals - Instead of opening a new modal on top of the existing one, the existing window should stay open, and the content should change to display this confirmation after user clicks Submit. 
- [x] Modals - Use the same styling as the existing modal for this confirmation modal. 
- [x] Content - Add a line break after the first sentence of the description text (after "submitted successfully").

**Comment by jaydenalchin** [comment]
Hi @KristenHu this is looking very nice! 
CC @bradystroud 

1. Please use the dark theme success toast as per https://fkhadra.github.io/react-toastify/introduction/
2. Move the toast notification top-right instead of top-center
3. Simplify the confirmation message a bit more e.g. "Form submitted. We'll be in contact as soon as possible"
4. If possible, left-justify the text inside the toast.

**Comment by KristenHu** [comment]
> Hi @KristenHu this is looking very nice! CC @bradystroud
> 
> 1. Please use the dark theme success toast as per https://fkhadra.github.io/react-toastify/introduction/
> 2. Move the toast notification top-right instead of top-center
> 3. Simplify the confirmation message a bit more e.g. "Form submitted. We'll be in contact as soon as possible"
> 4. If possible, left-justify the text inside the toast.

Hey @jaydenalchin (CC @bradystroud ),

All done, check please😊

FYI: I just adjust the style of the booking form on consulting page, I'll update the toast on the educational page after this version's toast gets passed.

**Comment by jaydenalchin** [comment]
> Hey @jaydenalchin (CC @bradystroud ),
> 
> All done, check please😊


@KristenHu Test passed!  🎉

File: components/bookingButton/bookingButton.tsx [hunk 1]
```diff
@@ -22,6 +24,19 @@ export const BookingButton = ({
...
+      autoClose: 5000,
+      hideProgressBar: true,
+      closeOnClick: false,
+      pauseOnHover: true,
+      draggable: false,
+      theme: "dark",
+      pauseOnFocusLoss: false
+    });
+  }
+
```
Comment (bradystroud) [inline components/bookingButton/bookingButton.tsx]: Looks like this code is duplicated. Can we reuse it?

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Review by jaydenalchin — APPROVED** [review]

**Review by bradystroud — APPROVED** [review]
LGTM 🏄‍♂️

**Pull request merged**